app/src/main/res/layout/fragment_welcome_step1.xml

```diff
@@ -80,10 +80,12 @@
         android:gravity="center_horizontal"
         android:orientation="vertical">
 
-        <Spinner
+        <org.gdg.frisbee.android.view.AutoCompleteSpinnerView
           android:id="@+id/chapter_spinner"
-          android:layout_width="wrap_content"
-          android:layout_height="wrap_content" />
+          android:gravity="center"
+          android:layout_width="match_parent"
```

Comment:
I think `keyline1` or even `keyline2` margin at the left and right with `match_parent` would be good.